site/docs/v1/tech/messengers/message-templates.adoc

```diff
@@ -0,0 +1,69 @@
+---
+layout: doc
+title: Message Templates
+description: Configuring a Message Template
+---
+
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Adjust a Message Template
+
+A message template allows you to customize how your messenger communicates with your audience. You can add custom styles and text using FreeMarker.
+
+A new template can be added by click on the icon:plus[type=fas, role=ui-button green] Add button
+
+image::messengers/message-templates-home.png[Message Templates Home , width=1200]
+
+=== Add Message Template
+
+image::messengers/message-templates-add.png[Message Templates Add, width=1200]
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+A unique UUID is auto generated if left blank.
+
+[field]#Name# [required]#Required#::
+For display only. Name of the message template.
+
+[field]#Type# [optional]#Optional#::
+The type of message template. Options include: SMS, Kafka, and Generic.
+
+[field]#Default Text# [required]#Required#::
+The input accepts FreeMarker input and will allow crafting a custom template.
+
+=== Add Message Template - Localization
+
+image::messengers/message-templates-add-locals.png[Message Templates Add Locals, width=1200]
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+You can enter localization options here. See our guide to localization link:/docs/v1/tech/themes/localization[here]
```

Comment:
You need id, name and type here. (Type will always be SMS right now.)

Reply:
should be fixed with new screen shots now